apprise/plugins/NotifyFreeMobile.py

```diff
@@ -0,0 +1,207 @@
+
+# the URL will look something like this:
+#      https://smsapi.free-mobile.fr/sendmsg?msg=content
+#
+
+import requests
+from json import dumps
+
+from .NotifyBase import NotifyBase
+from ..common import NotifyType
+from ..AppriseLocale import gettext_lazy as _
+
+
+class NotifyFreeMobile(NotifyBase):
+    """
+    A wrapper for Free-Mobile Notifications
+    """
+
+    # The default descriptive name associated with the Notification
+    service_name = _('Free-Mobile')
+
+    # The services URL
+    service_url = 'https://mobile.free.fr/'
+
+    # The default secure protocol
+    secure_protocol = 'freemobile'
+
+    # A URL that takes you to the setup/help of the specific protocol
+    setup_url = 'https://github.com/caronc/apprise/wiki/Notify_freemobile'
+
+    # Plain Text Notification URL
+    notify_url = 'https://smsapi.free-mobile.fr/sendmsg'
+
+    # Define object templates
+    templates = (
+        '{schema}://{user}@{password}',
+    )
+
+    # The title is not used
+    title_maxlen = 0
+
+    # SMS Messages are restricted in size
+    body_maxlen = 160
+
+    # Define our tokens; these are the minimum tokens required required to
+    # be passed into this function (as arguments). The syntax appends any
+    # previously defined in the base package and builds onto them
+    template_tokens = dict(NotifyBase.template_tokens, **{
+        'user': {
+            'name': _('Username'),
+            'type': 'string',
+        },
+        'password': {
+            'name': _('Password'),
+            'type': 'string',
+            'private': True,
+        },
+    })
+
+    def __init__(self, **kwargs):
+        """
+        Initialize Free Mobile Object
+        """
+        super().__init__(**kwargs)
+
+        if not (self.user and self.password):
+            msg = 'A FreeMobile user and password ' \
+                  'combination was not provided.'
+            self.logger.warning(msg)
+            raise TypeError(msg)
+
+        return
+
+    def url(self, privacy=False, *args, **kwargs):
+        """
+        Returns the URL built dynamically based on specified arguments.
+        """
+
+        # Prepare our parameters
+        params = self.url_parameters(privacy=privacy, *args, **kwargs)
+
+        return '{schema}://{user}@{password}/?{params}'.format(
+            schema=self.secure_protocol,
+            user=self.user,
+            password=self.pprint(self.password, privacy, safe=''),
+            params=NotifyFreeMobile.urlencode(params),
+        )
+
+    def send(self, body, title='', notify_type=NotifyType.INFO, **kwargs):
+        """
+        Send our notification
+        """
+
+        # prepare our headers
+        headers = {
+            'User-Agent': self.app_id,
+        }
+
+        # Prepare our payload
+        payload = {
+            'user': self.user,
+            'pass': self.password,
+        }
+
+        # Our Message
+        params = {
+            'msg': body
```

Comment:
I think the msg should be in the payload instead of params.
It does not work for me like this.